### DISCUSSION TITLE
sqlitestudio startup processing

### DISCUSSION THREAD

**Discussion by vanderdoes** [pawelsalawa/sqlitestudio#4990] [General]
Hi,

I'm running 3.4.4 on Windows 10 and I imported a file with about 100,000 inserts into a stable.  SQLiteStudio gave a message about contents of editor are huge and disabled some highlighting.  However, from then on it locks up.  I have cancelled out of the program but on getting back in it must pull in the file again and locks up.  I've looked at start up parameters, but don's see one that cancels initial processing.  I uninstalled and reinstalled but it does the same thing.

How do I get it to cancel the startup processing of my sessions?

Thanks,
Jeff

**Comment by pawelsalawa** [comment]
1. Close the app.
2. Delete configuration file (which holds your session to be restored). Configuration file location: https://github.com/pawelsalawa/sqlitestudio/wiki/Configuration_directory_location
3. Start the app.
